Wikipedia/Networking/Fetchers/SavedArticlesFetcher.h

```diff
@@ -17,14 +23,14 @@
 + (SavedArticlesFetcher*)sharedInstance;
 + (void)                 setSharedInstance:(SavedArticlesFetcher*)fetcher;
 
-@property (nonatomic, strong, readonly) MWKSavedPageList* savedPageList;
-@property (nonatomic, strong, readonly) MWKDataStore* dataStore;
-
-- (instancetype)initWithDataStore:(MWKDataStore*)dataStore;
+@property (nonatomic, strong) MWKSavedPageList* savedPageList;
```

Comment:
This never changes right? Shouldn't just be read only and be passed in on initialization?